notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        for i in range(n_itr):
</pre><p>You should use <code>jax.lax.scan()</code> now probably you know how to use that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>